Library/Formula/pcs.rb

```diff
@@ -0,0 +1,38 @@
+class Pcs < Formula
+  desc "The terminal utility for Baidu Network Disk."
+  homepage "https://github.com/GangZhuo/BaiduPCS/"
+  url "https://github.com/GangZhuo/BaiduPCS/archive/0.2.3.tar.gz"
+  sha256 "e0a72e41560df3f1499cf5baf29404465d34b14a9bff82095335dd29f0a0216f"
+
+  head "https://github.com/GangZhuo/BaiduPCS.git"
+
+  depends_on "openssl"
+
+  def install
+    system "make", "clean"
+    system "make"
+    bin.install "bin/pcs"
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
These caveats don't seem to be Homebrew specific and can be removed. People should check upstream for general usage.